publishexe.targets

```diff
@@ -38,6 +37,11 @@
   <Target Name="FixUpCoreConsoleNames">
     <Move SourceFiles="$(Outdir)$(AssemblyName).exe" DestinationFiles="$(Outdir)$(AssemblyName).dll" />
     <Move SourceFiles="$(Outdir)CoreConsole.exe" DestinationFiles="$(Outdir)$(AssemblyName).exe" />
+    <!-- create a marker indicating that the file was renamed, this will be used during signing to sign the correct file -->
+    <Touch Files="$(Outdir)$(AssemblyName).publishedexe" AlwaysCreate="true" />
+    <ItemGroup>
```

Comment:
Why do these need to be special? Can we not just put the correct marker file and include the actual file extension as part of the marker file name? So to get the right file signed we just rename the signing file as well.